plat/xen/include/xen/arch-x86/xen.h

```diff
@@ -193,26 +181,6 @@ struct vcpu_guest_context {
     /* NB. User pagetable on x86/64 is placed in ctrlreg[1]. */
     unsigned long ctrlreg[8];               /* CR0-CR7 (control registers)  */
     unsigned long debugreg[8];              /* DB0-DB7 (debug registers)    */
-#ifdef __i386__
-    unsigned long event_callback_cs;        /* CS:EIP of event callback     */
-    unsigned long event_callback_eip;
-    unsigned long failsafe_callback_cs;     /* CS:EIP of failsafe callback  */
-    unsigned long failsafe_callback_eip;
-#else
-    unsigned long event_callback_eip;
```

Comment:
This is in an else case for __i386__, so everything else to x86. It should stay, right?

Reply:
yes, it should stay. sorry for the mistake, I accidentally removed it.